### PR TITLE
DS-395 [Remove] "bolt-theme()" Deprecated

### PR DESCRIPTION
**Pull request by MarcinMr**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-395

## Summary

All instances of `bolt-theme()` should be deleted.

## Details

- `Tools themes` folder was removed
- `Settings themes` folder was removed
- `Themes` folder was removed
- ` CSS vars` folder was removed
- No more instance of `bolt-theme()` exist on a site property.
- All usage of `--bolt-theme-*` CSS vars are converted to the new `--m-bolt-*` vars, [Vars reference is here](https://boltdesignsystem.com/pattern-lab/?p=visual-styles-theming-system).

## How to test

Check if there are no more `bolt-theme()` instances in the repository and if `--bolt-theme-*` CSS vars were converted to the new `--m-bolt-*` vars properly.

## Release notes

`Bolt-theme()` will be removed

### CODE REVIEW

File: docs-site/src/pages/pattern-lab/_patterns/40-components/button-deprecated/-60-button-with-3rd-party-js.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% include '@pl/00-deprecated-button-docs.twig' %}
 
-{% set themes = get_data('@bolt/themes.schema.json') %}
+{% set themes = ['dark', 'light'] %}
```
Comment (colbytcook) [inline docs-site/src/pages/pattern-lab/_patterns/40-components/button-deprecated/-60-button-with-3rd-party-js.twig:3]: Is it worth just removing this from the file, it doesn't seem to be used

Comment (mikemai2awesome) [reply docs-site/src/pages/pattern-lab/_patterns/40-components/button-deprecated/-60-button-with-3rd-party-js.twig:3]: I am in favor of removing this page altogether.

Comment (colbytcook) [reply docs-site/src/pages/pattern-lab/_patterns/40-components/button-deprecated/-60-button-with-3rd-party-js.twig:3]: @MarcinMr per @mikemai2awesome suggestion, lets remove this page

Comment (colbytcook) [reply docs-site/src/pages/pattern-lab/_patterns/40-components/button-deprecated/-60-button-with-3rd-party-js.twig]: @MarcinMr and @mikemai2awesome I just removed this page

**Review by colbytcook — APPROVED** [review]

**Pull request merged**